src/main/webapp/app/admin/health/health.component.html

```
@@ -56,15 +57,15 @@ <h5 *adminTitleBarTitle id="health-page-heading" class="mb-0" jhiTranslate="heal
                             </td>
                         </tr>
                     }
-                    <tr id="websocketConnection" class="border-b border-surface-100 odd:bg-surface-50 dark:border-surface-700 dark:odd:bg-surface-800">
-                        <td class="py-2 px-3">Websocket Connection (Client -> Server)</td>
-                        <td class="py-2 px-3 text-center" data-testid="status-cell">
+                    <tr id="websocketConnection">
+                        <td>Websocket Connection (Client -> Server)</td>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)